.yarnrc

```diff
@@ -1,2 +0,0 @@
-version-tag-prefix "releases/typescript/v"
-version-git-message "NPM package v%s"
```

Comment:
We don't appear to be using this feature of Yarn in recent package releases, so it is safe to remove.

E.g. https://github.com/foxglove/schemas/pull/156